backend/models/sequence.js

```diff
@@ -144,12 +264,43 @@ Sequence.getList = async (account, model, branch, revision, cleanResponse = fals
 };
 
 Sequence.updateSequence = async (account, model, sequenceId, data) => {
-	if (!data || !data.name || !utils.isString(data.name) || data.name === ""  || data.name.length >= 30) {
+	const toSet = {};
+
+	if (!data) {
 		throw responseCodes.INVALID_ARGUMENTS;
 	}
 
+	if (data.name) {
+		if (!utils.isString(data.name) || data.name === "" || data.name.length >= 30) {
+			throw responseCodes.INVALID_ARGUMENTS;
+		}
+
+		toSet.name = data.name;
+	}
+
+	if (data.revId) {
```

Comment:
might want to check for null explicitly, to allow the option for user to unset a revision should they wish